lang/src/program.rs

```diff
@@ -1,31 +1,44 @@
 use crate::error::ErrorCode;
 use crate::*;
 use solana_program::account_info::AccountInfo;
+use solana_program::bpf_loader_upgradeable::{self, UpgradeableLoaderState};
 use solana_program::instruction::AccountMeta;
 use solana_program::program_error::ProgramError;
 use solana_program::pubkey::Pubkey;
 use std::fmt;
 use std::ops::Deref;
 
+// TODO: can we remove _account?
+
```

Comment:
@armaniferrante ^

is there any use to this field, considering that it's going to be serialized to an empty struct I think all(?) of the time

Reply:
yes that sounds better to me. https://github.com/project-serum/anchor/issues/1140